services/rollout-service/pkg/datadogdora/datadogdora.go

```diff
@@ -0,0 +1,205 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com*/
+
+package datadogdora
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/DataDog/datadog-api-client-go/v2/api/datadog"
+	"github.com/DataDog/datadog-api-client-go/v2/api/datadogV2"
+	"github.com/freiheit-com/kuberpult/services/rollout-service/pkg/service"
+	"golang.org/x/sync/errgroup"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+type Config struct {
```

Comment:
Please give this a more specific name, e.g. `DoraConfig`

Reply:
ist this clear from the name of package? otherwise this stutters for auser of the package:
datadogdore.DoraConfig ?

Reply:
the same pattern is also used the revolution package